client/settings/settings-manager/index.js

```diff
@@ -2,16 +2,20 @@
 /**
  * External dependencies
  */
+import { Button, ExternalLink } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import React, { useState } from 'react';
-import { ExternalLink } from '@wordpress/components';
+import React from 'react';
+
 /**
  * Internal dependencies
  */
+import './style.scss';
+import { LoadableBlock } from '../../components/loadable';
+import { useSettings } from '../../data';
```

Comment:
```suggestion
import { LoadableBlock } from 'components/loadable';
import { useSettings } from 'data';
```

Reply:
There are more imports in this file that can be converted to absolute paths, but maybe this could be a start. What do you think?